### PR TITLE
Added Few DL Profs in Canada

### PR DESCRIPTION
**Pull request by quettabit**

### CODE REVIEW

File: faculty-affiliations.csv [hunk 1]
```diff
@@ -3357,6 +3359,7 @@ Gregory Dudek , McGill University
 Hamed Hatami , McGill University
 Jackie Chi Kit Cheung , McGill University
 Jérôme Waldispühl , McGill University
+Joelle Pineau , McGill University
```
Comment (emeryberger) [inline faculty-affiliations.csv]: Does not meet inclusion criteria as will be full-time employed elsewhere.

File: faculty-affiliations.csv [hunk 2]
```diff
@@ -8798,9 +8802,11 @@ Gena Hahn , University of Montreal
 Gilles Brassard , University of Montreal
 Guy Lapalme , University of Montreal
 Houari A. Sahraoui , University of Montreal
+Ioannis (Yannis) Mitliagkas , University of Montreal
```
Comment (emeryberger) [inline faculty-affiliations.csv]: This is not the correct DBLP entry. Please ensure that all entries correspond to the faculty name(s) given at [DBLP](http://dblp.org).

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please check over all faculty names and correct commented entries. Thanks.

File: faculty-affiliations.csv (hunk 2 above)
Comment (quettabit) [reply faculty-affiliations.csv]: Sure ..

File: faculty-affiliations.csv (hunk 1 above)
Comment (quettabit) [reply faculty-affiliations.csv]: Fine ..

**Comment by quettabit** [comment]
@emeryberger have a look at the latest commit message. Acted on the requested changes. Thanks.

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Thanks!

**Pull request merged**